packages/katello/katello/backup.rb

```diff
@@ -260,10 +260,15 @@ def online_backup(database)
           puts "Backing up Pulp data... "
           matching = false
           until matching
+            backup_file = File.join(@dir, '.pulp.snar.backup')
+            FileUtils.cp File.join(@dir, '.pulp.snar'), backup_file if File.exist? File.join(@dir, '.pulp.snar')
             checksum1 = run_cmd("find . -printf '%T@\n' | md5sum")
             create_pulp_data_tar
             checksum2 = run_cmd("find . -printf '%T@\n' | md5sum")
             matching = (checksum1 == checksum2)
+            if File.exist? backup_file
+              matching ? FileUtils.rm(backup_file) : FileUtils.mv(backup_file, '.pulp.snar.backup'), File.join(@dir, '.pulp.snar'))
```

Comment:
Looks like there is a syntax error here?
```
/root/foreman-packaging/packages/katello/katello/backup.rb:270: syntax error, unexpected ',', expecting keyword_end (SyntaxError)
...kup_file, '.pulp.snar.backup'), File.join(@dir, '.pulp.snar'...
```

Reply:
I see `File.join(@dir, '.pulp.snar')` used a few times, that could be a variable

Reply:
@johnpmitsch fixed, thanks